Dockerfile

```
@@ -136,6 +139,22 @@ COPY --chown=$MAMBA_USER:$MAMBA_USER micro_environment.yml ${TETHYS_HOME}/tethys
 
 WORKDIR ${TETHYS_HOME}/tethys
 
+# Set the versions of Django, Channels, and Daphne if provided in environment.tyml and micro_environment.yml
+RUN if [ -n "$DJANGO_VERSION" ]; then \
+  sed -i "s/- django>=3.2,<6/- django==${DJANGO_VERSION}/" environment.yml micro_environment.yml; \
```

Can the version part of the django be found using a regular expression? Try this to see if it works:
```suggestion
  sed -i "s/- django.+/- django==${DJANGO_VERSION}/" environment.yml micro_environment.yml; \
```

---

Yes, it can be actually with the code below now the sed command find `- django`, and it replaces it with the `DJANGO_VERSION` arg variable, but it also ignores anything ` -django-` becuase there is some dependencies that contain `django` in the name followed by a `-`:
`sed -i "s/\s*- django[^-].*/  - django==${DJANGO_VERSION}/" environment.yml micro_environment.yml;`